packages/fxa-payments-server/src/components/PaymentErrorView/index.tsx

```diff
@@ -88,8 +95,18 @@ export const PaymentErrorView = ({
         </div>
 
         <div className="footer" data-testid="footer">
-          <ActionButton />
-          <TermsAndPrivacy plan={plan} />
+          {/* This error code means the subscription was created successfully, but
+          there was an error loading the information on the success screen. In this
+          case, we do not want a "Try again" or "Manage subscription" button. */}
+          {error.code !== 'fxa_fetch_profile_customer_error' ? (
+            <ActionButton data-testid={'error-view-action-button'} />
+          ) : null}
+          <PaymentLegalBlurb provider={undefined} />
```

Comment:
Probably don't need to be part of this patch, but assuming `provider` here meaning payment provider, I think we can get that info to this scope.

Reply:
We discussed this, and we don't have a `customer` to get the `customer.payment_provider` off of until the last function called, `fetchProfileAndCustomer` on form submit, so `customer` wouldn't be defined until most of the calls that could error have been made to this point.

There may be another way to get this information, but it isn't easily available, so I'm going to leave it for now. Also the [MVP designs](---https://www.figma.com/file/cOIAuHIIbrPbAlAIcpejNM/🎡MVP-Subscription-Optimization-Flow?node-id=1064%3A15109) show both Stripe and PayPal privacy policies on this screen.